[11] Moonshot/Moonshot/Views/Mission/Crew List/MissionCrewListView.swift

```diff
@@ -9,7 +9,7 @@ import SwiftUI
 
 struct MissionCrewListView: View {
 	
-	@ObservedObject
+	@EnvironmentObject
```

Comment:
This change could change the usage pattern of this view model, since the other ones were used as observed objects